tests/test_maplestory.py

```
@@ -3,33 +1,122 @@
+                "evasion": 0,
+                "pdef": 0,
+                "mdef": 0,
+                "accuracy_required": "0 (每少1級 +0)",
+            },
+            "maps": ["楓之島：嫩寶狩獵場Ⅱ"],
+            "drops": [
+                {
+                    "name": "綠髮帶",
+                    "type": "裝備",
+                    "link": "https://maplesaga.com/library/cn/permalink/equip/1002067",
+                    "img": "https://mai0313.github.io/artale-drop/image/%E7%B6%A0%E9%AB%AE%E5%B8%B6.png",
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)